src/main/java/Entity/Chat.java

```diff
@@ -0,0 +1,92 @@
+package Entity;
+
+import com.google.cloud.Timestamp;
+import java.util.Comparator;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.PrimaryKey;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Column;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Table;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.NotMapped;
+
+@Table(name = "Chat")
+public class Chat {
```

Comment:
Can this class (and other similar classes) be final?

Reply:
Done